tests/test_complex_questions.py

```diff
@@ -181,55 +191,31 @@ def check_invalid(tui, name, format, invalid_value, help=None, err="Invalid inpu
     tui.sendline("- I'd love it")
     tui.send(Keyboard.Esc + Keyboard.Enter)
     expect_prompt(tui, "choose_list", "str", help="You see the value of the list items")
-    deque(
-        map(
-            tui.expect_exact,
-            [
-                "first",
-                "second",
-                "third",
-            ],
-        )
-    )
+    tui.expect_exact("first")
+    tui.expect_exact("second")
+    tui.expect_exact("third")
```

Comment:
Do you agree with this change (and the same below)? I think it's better to reduce automation like loops etc. in tests and especially in assertions.

Reply:
These are just 2 ways to express the same thing, so I'd keep it as it was.